cypress/support/step-definitions/flat-table.js

```diff
@@ -0,0 +1,163 @@
+    .and('be.visible');
+  flatTableHeader(FOURTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(FIFTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(SIXTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(SEVENTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(EIGHT_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+});
+
+Then('FlatTable has nine row', () => {
+  flatTableHeadRow().should('have.length', 1);
+  flatTableBodyRow().should('have.length', 8);
+});
+
+Then('FlatTable has sevn colums', () => {
+  flatTableHeadRow().children().should('have.length', 7);
+});
+
+Then('{string} header cell has value {string}', (position, text) => {
+  switch (position) {
+    case 'first':
+      flatTableHeaderCell(FIRST_ELEMENT).should('have.text', text)
+        .and('be.visible');
+      break;
+    case 'second':
+      flatTableHeaderCell(SECOND_ELEMENT).should('have.text', text)
+        .and('be.visible');
+      break;
+    case 'third':
+      flatTableHeaderCell(THIRD_ELEMENT).should('have.text', text)
+        .and('be.visible');
+      break;
+    case 'fourth':
+      flatTableHeaderCell(FOURTH_ELEMENT).should('have.text', text)
+        .and('be.visible');
+      break;
+    case 'fifth':
+      flatTableHeaderCell(FIFTH_ELEMENT).should('have.text', text)
+        .and('be.visible');
+      break;
+    case 'sixth':
+      flatTableHeaderCell(SIXTH_ELEMENT).should('have.text', text)
+        .and('be.visible');
+      break;
+    case 'seventh':
+      flatTableHeaderCell(SEVENTH_ELEMENT).should('have.text', text)
+        .and('be.visible');
+      break;
+    default: throw new Error('There are only seven checkbox elements on the page');
+  }
+});
+
+Then('FlatTable {int} row contains proper inner context', (indexRow) => {
+  flatTabelCell(indexRow).eq(FIRST_ELEMENT).should('have.text', 'Soylent CorpJohn Doe')
+    .and('be.visible');
+  flatTabelCell(indexRow).eq(SECOND_ELEMENT).should('have.text', '12/12/20')
+    .and('be.visible');
+  flatTabelCell(indexRow).eq(THIRD_ELEMENT).should('have.text', '20/12/20')
+    .and('be.visible');
+  flatTabelCell(indexRow).eq(FOURTH_ELEMENT).should('have.text', '25/12/20')
+    .and('be.visible');
+  flatTabelCell(indexRow).eq(FIFTH_ELEMENT).should('have.text', 'business')
+    .and('be.visible');
+  flatTabelCell(indexRow).eq(SIXTH_ELEMENT).should('have.text', 'Group1, Group2, Group3')
+    .and('be.visible');
+  flatTabelCell(indexRow).eq(SEVENTH_ELEMENT).should('have.text', 'Accounting')
+    .and('be.visible');
+});
+
+Then('I scroll tabel content to right bottom', () => {
+  cy.viewport(625, 450);
+  flatTable().parent().scrollTo('100%', '100%');
+});
+
+Then('{string} header cell is not visible and has value {string}', (position, text) => {
```

Comment:
Do we need to put text here if it's not visible?
```suggestion
Then('{string} header cell is not visible', (position) => {
```

Reply:
fixed